server/@types/models/CourseParticipation.ts

```diff
@@ -25,8 +25,22 @@ type CourseParticipation = {
   source?: string
 }
 
+type CourseParticipationUpdate = {
+  outcome: CourseParticipationOutcome
+  setting: CourseParticipationSetting
```

Comment:
Would this allow us to send an `undefined` setting, given `type` is a compulsory setting property? (C.f. all of the outcome properties are optional)

Reply:
Looks like `{}` is how we're meant to send `undefined` settings/outcomes: https://mojdt.slack.com/archives/C05045L6MEE/p1696001001594769

Reply:
Oh yeah, we should update the `type` to be optional now. I'll add a card and open a PR for a couple of small fixes today

Reply:
A participation's `type` is now optional?